releases.yml

```
@@ -1,4 +1,4 @@
-- version: 4.8.0
+- version: 4.8.1
```

If we are not adding anything to changelog we should not change the version (especially that it's the one already released).

So please revert this change.

```suggestion
- version: 4.8.0
```